ui/src/router/details/components/router_details_header/RouterDetailsPageHeader.scss

```diff
@@ -1,3 +1,8 @@
+.euiFlexGroup--justifyContentSpaceBetween {
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
```

Comment:
You should use `EuiFlexGroup` props for this, instead of custom styles: https://elastic.github.io/eui/#/layout/flex#flex-group-is-for-a-single-row-layout
```js
<EuiFlexGroup justifyContent="spaceBetween" wrap={true}>
...
</EuiFlexGroup>
```